api/src/services/images_service.rs

```diff
@@ -68,27 +67,26 @@ impl ImagesService {
                 }
             };
 
-            if let Some((config, blob)) = blob {
+            if let Some((image, blob)) = blob {
                 if let Some(port) = blob.get_exposed_port() {
-                    port_mappings.insert(config.clone(), port);
+                    port_mappings.insert(image.clone(), port);
                 }
             }
         }
 
         Ok(port_mappings)
     }
 
-    async fn resolve_image_blob<'a>(
-        config: &'a ServiceConfig,
-        image: &Image,
-    ) -> Result<Option<(&'a ServiceConfig, ImageBlob)>, ImagesServiceError> {
-        debug!("Resolve image manifest for {}", config.image().to_string());
+    async fn resolve_image_blob(
+        img: &Image,
```

Comment:
Hmm, in other places, `image` is introduced as a name, but here it is renamed to `img`. If this is to avoid Confusion with the `image` below — maybe rename that to `image_name`?

Reply:
Done.